variables.tf

```diff
@@ -150,6 +150,19 @@ variable "master_additional_files" {
   type        = list(string)
 }
 
+variable "master_additional_labels" {
+  description = "List of labels to append to `role=master` for master nodes. Labels are expected in <key=label> format by kubelet."
+  type        = map(string)
+  # example default labels:
```

Comment:
Being very picky here, I'm not sure we need an example, its just a simple Terraform map.